plugins/serialize-disabled-interactions/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@blockly/plugin-serialize-disabled-interactions",
```

Comment:
Could you add this to the [blockly-samples README](https://github.com/google/blockly-samples/tree/master/plugins#readme) and [index.md ](https://github.com/google/blockly-samples/blob/master/gh-pages/index.md)?